package.json

```diff
@@ -13,11 +13,15 @@
     "@pollyjs/adapter-fetch": "^2.4.0",
     "@pollyjs/adapter-xhr": "^2.4.0",
     "@pollyjs/persister-rest": "^2.1.0",
+    "antd": "^3.21.4",
+    "babel-plugin-import": "^1.12.0",
     "babel-polyfill": "^6.26.0",
     "classnames": "^2.2.6",
+    "customize-cra": "^0.5.0",
     "lodash": "^4.17.11",
     "prop-types": "^15.7.2",
     "react": "^16.8.6",
+    "react-app-rewired": "^2.1.3",
     "react-dom": "^16.8.6",
     "react-scripts": "^3.1.0",
```

Comment:
Can we remove this package as we are using react-app-rewired?

Reply:
I don't quite get what you're saying, without this package modules necessary for react-app-rewired won't be installed therefore the app won't run. Could you expand your answear a bit more?

Reply:
I mean to remove react-scripts as we are no longer using it